spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,132 @@
+require_relative '../lib/game'
+require_relative '../lib/board'
+require_relative '../lib/player'
+
+module TicTacToe
+  describe Game do
+    let(:mockPlayer) { double(player1: 'Player1', player2: 'Player2') }
+    let(:game) { Game.new(mockPlayer) }
+    describe '.initialize' do
+      it 'return the name of the players' do
+        expect(game.player.player1).to eq 'Player1'
+      end
+      it 'raise error if there is no arguments' do
+        expect { Game.new }.to raise_error
+      end
+    end
```

Comment:
- [ ] Using the `raise_error` matcher without providing a specific error or message risks false positives, since `raise_error` will match when Ruby raises a `NoMethodError`, `NameError` or `ArgumentError`, potentially allowing the expectation to pass without even executing the method you are intending to call. Instead, consider providing a specific error class or message.